spec/support/vcr_cassettes/kafka/topics.yml

```diff
@@ -0,0 +1,125 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: http://localhost:8081/subjects/topic-1/versions/latest
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      Accept:
+      - application/json
+      Content-Type:
+      - application/json
+      User-Agent:
+      - Vets.gov Agent
+      Referer:
+      - https://review-instance.va.gov
+      X-Vamf-Jwt:
+      - stubbed_token
+      X-Request-Id:
+      - ''
+      Accept-Encoding:
+      - gzip;q=1.0,deflate;q=0.6,identity;q=0.3
+  response:
+    status:
+      code: 200
+      message: OK
+    headers:
+      Date:
+      - Tue, 10 Aug 2021 15:55:41 GMT
+      Content-Type:
+      - application/json
+      Transfer-Encoding:
+      - chunked
+      Server:
+      - openresty
+      X-Vamf-Version:
+      - 1.9.0
+      B3:
+      - e1836f37ef12a997-bd3cba28a9b29067-1
+      Access-Control-Allow-Headers:
+      - x-vamf-jwt
+      X-Vamf-Build:
+      - e208742
+      X-Vamf-Timestamp:
+      - '2021-08-03T18:59:01+0000'
+      Access-Control-Allow-Origin:
+      - "*"
+      Access-Control-Allow-Methods:
+      - GET,OPTIONS
+      Access-Control-Max-Age:
+      - '3600'
+      Strict-Transport-Security:
+      - max-age=63072000; includeSubDomains; preload
+    body:
+      encoding: UTF-8
+      string: '{
+                "subject": "submission_trace_mock_dev-value",
```

Comment:
This is just me being pedantic but because of the naming conventions that exist between topic, schema registry url, subject, etc., we might want to rename this to `topic-1-value`.

Reply:
yeah, that's a good point, updated.